src/xtgeo/grid3d/_grid3d_fence.py

```diff
@@ -149,18 +150,18 @@ def _update_tmpvars(self: Grid, force: bool = False) -> None:
 def _get_randomline_fence(
     self: Grid,
     polygon: Polygons,
-    hincrement: float | int | None,
+    distance: float | int | None,
     atleast: int,
     nextend: int,
 ) -> np.ndarray:
     """Compute a resampled fence from a Polygons instance."""
-    if hincrement is None:
+    if distance is None:
+        logger.info("Estimate hincrement from Polygons instance...")
         geom = self.get_geometrics()
 
         avgdxdy = 0.5 * (geom[10] + geom[11])
         distance = 0.5 * avgdxdy
-    else:
-        distance = hincrement
+        logger.info("Estimate hincrement from Polygons instance... DONE")
```

Comment:
Maybe these log statements should go to `logger.debug` instead? They also don't reflect hincrement renamed to distance